### DISCUSSION TITLE
Help needed web app

### DISCUSSION THREAD

**Discussion by Fds36bon** [CardinalKit/CardinalKit#163] [General]
Hi there I carnt get the web app to work there are some missing files could you help please. What’s your email

**Comment by vishnuravi** [comment]
We are currently reworking our web dashboard and it is not currently available for download. We will provide updates soon. Sorry for the inconvenience.

**Reply by Fds36bon** [reply]
You mention early 2023 on your website would this be in the next few weeks?. 
could you give guidance I have data in firebase storage which is collected by the app, how do I download the data to a excel sheet could you write some steps or guidance. Can I download the old web dashboard from your website and use that. It would be great if you could release the web dashboard so we can extract the data from firebase to excel and do analysis

**Reply by Fds36bon** [reply]
Thabkyou so much for taking the time to read my request and Thankyou so much for your work

**Reply by vishnuravi** [reply]
Hi @Fds36bon please send us an email at cardinalkit@stanford.edu so we can figure out how to best help you. 

Thanks for using CardinalKit!

**Reply by Fds36bon** [reply]
Thankyou for your quick response I have emailed you any help will be much appreciated I have been stuck on this for couple of weeks now. I would be very greatful

Thankyou

**Comment by feraco** [comment]
Join the slack group in might be easier to help.

**Reply by Fds36bon** [reply]
I get a error saying slack link doesn’t work
